docs/source/query-plans.mdx

```diff
@@ -436,6 +438,59 @@ When the router sees this special `Fetch` syntax, it knows to query a subgraph's
 
 Now that you've learned about each query plan node, take another look at the example query plan in [Example graph](#example-graph) to see how these nodes work together in a complete query plan.
 
+### Defer node
+
+A `Defer` node corresponds to one or more `@defer`s at the same level of nesting in the query plan.
+
+The node contains a _primary block_ and an array of _deferred blocks_. The primary block represents the part of the query that isn't deferred. Each deferred block corresponds to the one deferred part of the query.
+
+Read more about how `@defer` works in the [Apollo Router support for @defer article](/router/executing-operations/defer-support/).
+
+```graphql
+QueryPlan {
+  Defer {
+    Primary {
+      Fetch(...) {}
+    }, [
+      Deferred(...) {
+        Flatten(...) {
+          Fetch(...) {}
+        }
+      }
+    ]
+  }
+}
+```
+
+### Condition node
+
+A `Condition` node splits a query plan into an if-else branch. `Condition` nodes are commonly used for `@skip` and `@include` directives so the query plan can select different nodes based on the provided runtime variables.
+
+```graphql
+QueryPlan {
+  Sequence {
+    Fetch(...) {}
+    Include(...) {
+      Flatten(...) {
+        Fetch(...) { }
+      }
+    }
+  }
+}
+```
+```graphql
+QueryPlan {
+  Sequence {
+    Fetch(...) {}
+    Skip(...) {
+      Flatten(...) {
+        Fetch(...) { }
+      }
+    }
+  }
+}
```

Comment:
I don't see a `Condition` node in the current example. Is this correct?:

```suggestion
QueryPlan {
  Condition(if: $condition) {
    Then {
      Sequence {
        Fetch(...) {}
        Include(...) {
          Flatten(...) {
            Fetch(...) { }
          }
       }
    }  Else {
       Sequence {
         Fetch(...) {}
         Skip(...) {
           Flatten(...) {
             Fetch(...) { }
          }
       }
    }
 }
```

Reply:
Yes, I will update the doc header. We renamed the node in a recent Federation version which I generated the example query plans from